web/.eslintrc.json

```diff
@@ -20,6 +20,25 @@
     "prettier"
   ],
   "rules": {
+    "import/order": [
+      "error",
+      {
+        "groups": ["external", "internal"],
+        "pathGroups": [
+          {
+            "pattern": "react",
+            "group": "external",
+            "position": "before"
+          }
+        ],
```

Comment:
Should we add a group for "node*" imports as well? Would be nice to group all the internal stuff seperate from our code imports.